composer.json

```diff
@@ -24,7 +24,7 @@
 		"GPL-2.0-only"
 	],
 	"require" : {
-		"php" : ">=7.1",
+		"php" : ">=7.1 <8.1",
```

Comment:
are we sure that it's valid notation?
also why we need this upper restriction?

Reply:
Apparently we need to work on 8.1 support here a bit more.
When I added php 8.1-powered CI, it failed [here](https://github.com/oat-sa/qti-sdk/runs/6877817959?check_suite_focus=true).